channel/consensus_channel/leader_channel_test.go

```diff
@@ -228,6 +228,7 @@ func TestLeaderChannel(t *testing.T) {
 		c := testChannel(startingOutcome, startingQueue)
 
 		newAdd := Proposal{ToAdd: add(2, amountAdded, types.Destination{3}, alice, bob)}
+		newAdd.ChannelID = p1.ChannelID
```

Comment:
nit but could we set `ChannelID` when initialing `Proposal` in the line above?

Reply:
6dd1662f2ce6c90b483addd120f458590a42cd14